### PR TITLE
OHRI-966 OHRI-967 Feature/ohri 511 care and treatment s27 d20

### PR DESCRIPTION
**Pull request by larslemos**
This PR covers the below: 

Screen Shot 2022-08-11 at 10 19 38

### CODE REVIEW

File: packages/esm-hiv-app/src/constants.ts [hunk 1]
```diff
@@ -156,3 +156,7 @@ export const ViralLoadResultDate_UUID = '163724AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA';
 export const ReasonForViralLoad_UUID = '86cc0cfe-bace-4969-94b6-d139f4971d13';
 export const ViralLoadResult_UUID = '1305AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA';
 export const ViralLoadCopies_UUID = '856AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA';
+
+//Care and Treatement
+export const enrolmentDate = '160555AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA';
+export const dateOfARtIniation = '159599AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA';
```
Comment (pirupius) [inline packages/esm-hiv-app/src/constants.ts:162]: this looks like a typo

File: packages/esm-hiv-app/src/views/service-summary/encounter-list/service-summary-encounter-list.component.tsx [hunk 2]
```diff
@@ -1,20 +1,136 @@
...
+      },
+    },
+    //TODO: Refactor to include in within VL above
+    // {
+    //   key: 'vlDate',
+    //   header: t('vlDate', 'Recent VL Date'),
+    //   getValue: encounter => {
+    //     return getObsFromEncounter(encounter, ViralLoadResult_UUID);
+    //   },
+    // },
```
Comment (pirupius) [inline packages/esm-hiv-app/src/views/service-summary/encounter-list/service-summary-encounter-list.component.tsx:52]: do we still need the commented code?

File: packages/esm-hiv-app/src/views/service-summary/encounter-list/service-summary-encounter-list.component.tsx [hunk 3]
```diff
@@ -1,20 +1,136 @@
...
+      },
+    },
+    //TODO: Refactor to include in within CD4 above
+    // {
+    //   key: 'cd4ResultDate',
+    //   header: t('cd4ResultDate', 'Recent CD4 Date'),
+    //   getValue: encounter => {
+    //     return getObsFromEncounter(encounter, Cd4LabResultDate_UUID, true);
+    //   },
+    // },
```
Comment (pirupius) [inline packages/esm-hiv-app/src/views/service-summary/encounter-list/service-summary-encounter-list.component.tsx:67]: same comment as above. Are concepts missing?

File: packages/esm-hiv-app/src/constants.ts (hunk 1 above)
Comment (larslemos) [reply packages/esm-hiv-app/src/constants.ts:162]: Fixed Typo.

File: packages/esm-hiv-app/src/views/service-summary/encounter-list/service-summary-encounter-list.component.tsx (hunk 2 above)
Comment (larslemos) [reply packages/esm-hiv-app/src/views/service-summary/encounter-list/service-summary-encounter-list.component.tsx:52]: Yes, this is a Todo. 
Still going to work on it today evening and close

**Pull request merged**